### DISCUSSION TITLE
How can I disable the ripple effect of button globally?

### DISCUSSION THREAD

**Discussion by mxvsh** [heroui-inc/heroui#2267] [Help] [answered]
I couldn't find anything on docs for disabling ripple effect globally for every component.

**Comment by mxvsh** [comment]
anyone?

**Comment by wangwailok** [comment]
I wanna know as well

**Comment by alphaxek** [comment]
@wangwailok @mxvsh do you want to disable it in whole project once or you want to toggle between disable and unable?

**Comment by wingkwong** [comment, marked as answer]
in `NextUIProvider`, you can set `disableRipple`  to `true`. See https://nextui.org/blog/v2.4.0#disableripple

**Comment by alphaxek** [comment]
@mxvsh is your issue resolved?

**Comment by bryanherediax** [comment]
v2.4.0 supports disabling animations globally. 

https://nextui.org/blog/v2.4.0#disable-animations-globally
